tests/pytest/tests.sh

```diff
@@ -49,9 +50,39 @@ install_git_lfs() {
 
 #----------------------------------------------------------------------------------------------
 
-check_redis_server() {
-	if ! command -v redis-server > /dev/null; then
-		echo "Cannot find redis-server. Aborting."
+setup_redis_server() {
+	if [[ -n $SAN ]]; then
```

Comment:
${SAN} (etc) makes for safer shell scripts

Reply:
So you'd like to wrap any variable reference with {}, even when it's clear we're on the safe side? Wouldn't that hurt readability a bit? BTW I'm more concerned with quoting (in this case bash handles this ok so it can be omited, but in too many cases you do see `A=$B`).